packages/assets-controllers/src/index.ts

```diff
@@ -147,3 +147,21 @@ export type {
   RatesControllerPollingStartedEvent,
   RatesControllerPollingStoppedEvent,
 } from './RatesController';
+export {
+  BalancesTracker,
+  MultichainBalancesController,
```

Comment:
do we need to be explicit here or can we just use a wildcard export like `export * from ....`

Reply:
The entire file is explicit, I rather follow the same pattern